interlok-core/src/main/java/com/adaptris/core/AdaptrisMessageImp.java

```diff
@@ -413,18 +413,22 @@ private String resolve(String s, Pattern pattern) {
    */
   @Override
   public Object resolveObject(String s) {
+    if (s == null) {
```

Comment:
@albinoloverats it's probably better to check all the test coverage warnings and add tests where appropriated such as for AdaptrisMessageImp#resolveObject for line 416 and 417.